ERC721/serverless-NFT/1-06.md

```diff
@@ -0,0 +1,208 @@
+
+##### Step 6: Make Tests
+
+Rename `Sample.test.js` to `Token.test.js` or whatever your token contract is called, then replace all references to "Sample" with "Token".
+
+```bash
+mv ./test/Sample.test.js ./test/Token.test.js
+```
+
+
+
+```javascript
+var Token = artifacts.require('./Token.sol')
+var BigNumber = require('bignumber.js')
+let gasPrice = 1000000000 // 1GWEI
+
+let _ = '        '
+
+contract('Token', async function(accounts) {
+  let token
+
+  before(done => {
+    ;(async () => {
+      try {
+        var totalGas = new BigNumber(0)
+
+        // Deploy Token.sol
+        token = await Token.new()
+        var tx = await web3.eth.getTransactionReceipt(token.transactionHash)
+        totalGas = totalGas.plus(tx.gasUsed)
+        console.log(_ + tx.gasUsed + ' - Deploy Token')
+        token = await Token.deployed()
+
+        console.log(_ + '-----------------------')
+        console.log(_ + totalGas.toFormat(0) + ' - Total Gas')
+        done()
+      } catch (error) {
+        console.error(error)
+        done(false)
+      }
+    })()
+  })
+
+  describe('Token.sol', function() {
+    it('should pass', async function() {
+      assert(
+        true === true,
+        'this is true'
+      )
+    })
+
+  })
+})
+```
+
+Add the Metadata file and repeat the process in the migrations file.
+
+Also don't forget to set the deploy parameters for the token, including the metadata address.
```

Comment:
@okwme What does this line mean? What are you supposed to add and where?

Reply:
ah that's referring to this line:
```
        token = await Token.new("Token", "TKN", metadata.address)
```
The deploy parameters are the token name, symbol and metadata address.